usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -111,13 +111,13 @@ def signal_handler(signal, frame):
         logger.info("Getting File C data")
         # we dont have sub-tier agency info, so we'll do our best
         # to match them to the more specific award records
-        award_financial_query = "SELECT * FROM certified_award_financial WHERE submission_id = {0}".format(
-            submission_id
+        award_financial_query = (
+            "SELECT * FROM certified_award_financial"
+            f" WHERE submission_id = {submission_id}"
+            " AND transaction_obligated_amou IS NOT NULL AND transaction_obligated_amou != 0"
         )
-        if isinstance(db_cursor, PhonyCursor):  # spoofed data for test
-            award_financial_frame = pd.DataFrame(db_cursor.db_responses[award_financial_query])
-        else:  # real data
-            award_financial_frame = pd.read_sql(award_financial_query, connections["data_broker"])
```

Comment:
I'm all for getting rid of test hooks when we can, but are we sure this wasn't used anywhere?

Reply:
Was only be used by the `load_submission` test I edited from what I noticed. With Broker integration tests we should be able to move away from mocking the DB Cursor.